src/bandwidthRtc.ts

```diff
@@ -91,7 +101,10 @@ class BandwidthRtc {
     const peerConnection = new RTCPeerConnection(RTC_CONFIGURATION);
     this.setupNewPeerConnection(peerConnection, endpointId, mediaTypes);
     mediaStream.getTracks().forEach((track) => {
-      peerConnection.addTrack(track, mediaStream);
+      var sender = peerConnection.addTrack(track, mediaStream);
+      if (track.kind === "audio") {
+        this.dtmfSender = new DTMFSender(sender);
```

Comment:
This is a really good point. I think the code as-is would actually break if someone published a second stream and then unpublished it, because the DtmfSender would still be associated with the now nonexistent MediaStream. We probably need to have a separate DtmfSender for each audio stream that is published, like Chris suggested, and manage its lifecycle with that of the stream itself.

Reply:
Good points. I'll test and validate behavior and report back with results so we can finalize the approach.

Reply:
Observations before the fix (with help from Nicko):
1. Two streams from the same browser
2. First stream loses audio